vernac/g_vernac.mlg

```diff
@@ -119,7 +119,8 @@ GRAMMAR EXTEND Gram
     ]
   ;
   attr_value:
-    [ [ "=" ; v = string -> { VernacFlagLeaf v }
+    [ [ "=" ; v = string -> { VernacFlagLeaf (FlagString v) }
+      | "=" ; v = IDENT -> { VernacFlagLeaf (FlagIdent v) }
```

Comment:
I wonder if it would make sense to take qualids here.
We don't have a use for it currently but plugins could have one and that way they wouldn't need to feature request + wait for the next Coq version.

Reply:
Should we keep the `IDENT` case then?

Reply:
no, we would have a qualid case instead